src/DotNetWorker.Core/Converters/Converter/IConverterContextFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Converters
+{
+    /// <summary>
+    /// An abstraction to create <see cref="ConverterContext"/> instances.
+    /// </summary>
+    public interface IConverterContextFactory
```

Comment:
Does this need to be public in this pass?

Reply:
Good catch. No, this can be `internal` for this version. Changed.